src/go/manifest.json

```
@@ -1,12 +1,10 @@
 {
-  "version": "1.2.0",
+  "version": "1.2.1",
```

```suggestion
  "version": "1.2.0",
```

For images we don't bump version in the PR, it's in an isolated one